dev-lisp/ecls/ecls-20.4.24.ebuild

```diff
@@ -48,13 +51,16 @@ pkg_setup() {
 src_prepare() {
 	default
 	cp "${EPREFIX}"/usr/share/common-lisp/source/asdf/build/asdf.lisp contrib/asdf/ || die
+	cd src
```

Comment:
|| die

Reply:
I added the `die` statement and I put a message to it so that it is distinctive.